extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/StaticTenantResolver.java

```diff
@@ -93,13 +96,16 @@ private DefaultStaticTenantResolver(TenantConfigBean tenantConfigBean) {
 
         @Override
         public String resolve(RoutingContext context) {
-            String[] pathSegments = context.request().path().split("/");
-            if (pathSegments.length > 0) {
-                String lastPathSegment = pathSegments[pathSegments.length - 1];
-                if (tenantConfigBean.getStaticTenant(lastPathSegment) != null) {
+            String path = context.request().path();
+            if (isNullOrEmpty(path)) {
```

Comment:
This check is unncessary, it can't be null and it is unlikely to ever be empty in any real application